sdk/confidentialledger/azure-security-confidentialledger/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.0.0 (2022-06-18)
```

Comment:
Is this a GA release, if yes we should probably mention it so https://github.com/Azure/azure-sdk-for-java/blob/19d39a2836111cd79431e43ce45e8fb7f549f29b/sdk/storage/azure-storage-common/CHANGELOG.md#L16

Needs sectioning for `New Features`,..

Reply:
Date needs to be updated.